charts/delphai-keda/templates/deployment.yml

```diff
@@ -68,6 +68,13 @@ spec:
             - name: {{ .name }}
               value: {{ .value }}
             {{- end }}
+            {{ if .Values.gpu }}
+            - name: NVIDIA_VISIBLE_DEVICES
+              value: "all"
+            - name: NVIDIA_DRIVER_CAPABILITIES
+              value: "compute,utility"
```

Comment:
Yes, it should be like that, check this https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/user-guide.html#driver-capabilities